apps/dsp-app/src/app/workspace/search/fulltext-search/fulltext-search.component.ts

```diff
@@ -210,15 +210,16 @@ export class FulltextSearchComponent implements OnInit, OnChanges, OnDestroy {
     getAllProjects(): void {
         this._dspApiConnection.admin.projectsEndpoint.getProjects().subscribe(
             (response: ApiResponseData<ProjectsResponse>) => {
-                this.projects = response.body.projects;
-                // this.loadSystem = false;
+                // filter out deactivated projects and system projects
+                this.projects = response.body.projects.filter(p => p.status === true && !this.doNotDisplay.includes(p.id));
```

Comment:
Maybe replace the name "doNotDisplay" with "hideProjects" so there is not double negation

Reply:
makes sense, changed in [bf146e0](https://github.com/dasch-swiss/dsp-das/pull/1015/commits/bf146e01f563b5c8fc7e0af664bf5b758c19cc80)